modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp

```diff
@@ -84,6 +86,10 @@ long formInstanceId = ddmFormDisplayContext.getFormInstanceId();
 					pageDescription = ddmFormDisplayContext.getSuccessPageDescription(displayLocale);
 					pageTitle = ddmFormDisplayContext.getSuccessPageTitle(displayLocale);
 				}
+				else if (expired) {
+					pageDescription = LanguageUtil.get(request, "this-form-has-an-expiration-date");
+					pageTitle = LanguageUtil.get(request, "this-form-is-no-longer-available");
+				}
```

Comment:
@rodrigopaulino Shouldn't we first check if the form is expired instead of showing the success page? I'm actually not familiar with the expected behavior.

Reply:
I don't think this is relevant because they are supposed to be excluding flags.

Reply:
Got it! Thanks, @rodrigopaulino!